tests/test_persistence.py

```diff
@@ -52,3 +52,44 @@ def test_libsecret_persistence(temp_location):
         {"my_attr_1": "foo", "my_attr_2": "bar"},
         ))
 
+@pytest.mark.skipif(
+    is_running_on_travis_ci, reason="Requires manual testing")
+def test_windows_empty_file_exists_before_first_use():
+    test_folder = tempfile.mkdtemp(prefix="msal_extension_test_windows_token_cache_roundtrip")
```

Comment:
Unlike other existing test files, the test_persistence.py already uses the `temp_location` approach which seems working well. Would recommend to use it here.